explorer/interpreter/interpreter.cpp

```diff
@@ -462,7 +469,8 @@ auto Interpreter::StepLvalue() -> ErrorOr<Success> {
       if (act.pos() == 0) {
         // Get LValue for expression.
         return todo_.Spawn(std::make_unique<LValAction>(&access.object()));
-      } else {
+      }
+      {
```

Comment:
Likely unintended as well
```suggestion
      } else {
```